torch/distributed/fsdp/flat_param.py

```diff
@@ -1943,7 +1943,7 @@ def _writeback_orig_params(self) -> bool:
                         flat_param_grad.untyped_storage().data_ptr()
                     )
         # TODO: If we want to handle shared parameters, we need to re-generate
-        # the shared parameter data structures in case sharedness changed.
+        # the shared parameter data structures in case shardness changed.
```

Comment:
This should be sharedness (even if that is not a real word). I meant the state of being "shared".

Reply:
@awgu thank you for your commented. I updated the branch

Reply:
Thanks!